src/components/SearchBox/onSelectedFactory.ts

```diff
@@ -110,10 +117,13 @@ export const onSelectedFactory =
         overpassOptionSelected(option, setOverpassLoading, bbox, showToast);
         break;
       case 'geocoder':
-        geocoderOptionSelected(option, setFeature);
+        geocoderOptionSelected(option, setFeature, addToHistoryOption);
```

Comment:
In the future, I would like all the `Option` types to be saved to history after they are selected. There is no reason why Preset or OsmOption shouldn't be offered as well.

Probably leave that to a followup PR. Maybe after https://github.com/zbycz/osmapp/issues/621 🙂